internal/app/osd/proto/api.proto

```diff
@@ -77,9 +77,61 @@ message RoutesReply { repeated Route routes = 1; }
 
 // The response message containing a route.
 message Route {
+
+  // Interface is the interface over which traffic to this destination should be sent
   string interface = 1;
+
+  // Destination is the network prefix CIDR which this route provides
   string destination = 2;
+
+  // Gateway is the gateway address to which traffic to this destination should be sent
   string gateway = 3;
+  
+  // Metric is the priority of the route, where lower metrics have higher priorities
+  uint32 metric = 4;
+
+  // Scope desribes the scope of this route
+  uint32 scope = 5;
+
+  // Source is the source prefix CIDR for the route, if one is defined
+  string source = 6;
+
+  // Family is the address family of the route.  Currently, the only options are AF_INET (IPV4) and AF_INET6 (IPV6).
+  AddressFamily family = 7;
+
+  // Protocol is the protocol by which this route came to be in place
+  RouteProtocol protocol = 8;
+
+  // Flags indicate any special flags on the route
+  uint32 flags = 9;
+}
+
+enum AddressFamily {
+   option allow_alias = true;
+   AF_UNSPEC = 0x0;
+   AF_INET = 0x2;
+   IPV4 = 0x2;
+   AF_INET6 = 0xa;
+   IPV6 = 0xa;
+}
+
+enum RouteProtocol {
+   RTPROT_UNSPEC = 0;
+   RTPROT_REDIRECT = 1;  // Route installed by ICMP redirects
+   RTPROT_KERNEL = 2;    // Route installed by kernel
+   RTPROT_BOOT = 3;      // Route installed during boot
+   RTPROT_STATIC = 4;    // Route installed by administrator
+   RTPROT_GATED = 8;	    // Route installed by gated
```

Comment:
This is a super nit 😄 : Comments aren't aligned, but it looks like you meant to have them aligned.

Reply:
Thanks; looks like some tabs remained from the copy-paste.